python/lsst/ctrl/pool/parallel.py

```diff
@@ -262,7 +262,9 @@ def submitCommand(self, scriptName):
         return "exec %s" % scriptName
 
 
-BATCH_TYPES = {'pbs': PbsBatch,
+BATCH_TYPES = {'none' : None,
+               'None' : None,
```

Comment:
I could do that, but wouldn't I also have to implement case-insensitive argument parsing for the validate?  I've done that elsewhere, but it's messy.